src/components/CodeBox.vue

```diff
@@ -1,29 +1,61 @@
 <template>
-  <br />
-  <span v-for="word in currentCode.words" v-bind:key="word">
-    <br v-if="word.word == newLine" />
-    <button v-if="word.word != tab && word.word != newLine" @click="clickedButton(word.id)">{{ word.word }}</button>
-  </span>
+  <div class="codebox">
+    <div v-for="line in codeLines" :key="line">
```

Comment:
It needs a key, if there is no key specified, it will not run
` 3:5  error    Elements in iteration expect to have 'v-bind:key' directives  vue/require-v-for-key`